src/core_atmosphere/mpas_atm_core.F

```diff
@@ -851,6 +855,9 @@ subroutine atm_do_timestep(domain, dt, itimestep)
       real (kind=RKIND) :: xtime_s
       integer :: ierr
 
+      clock => domain % clock
+      mpas_log_info => domain % logInfo
+
```

Comment:
There's no problem to have these assignments here, but just out of curiosity, does JEDI call `atm_do_timestep` outside of a call to `atm_core_run`? In stand-alone MPAS-A, the only call to `atm_do_timestep` is inside of `atm_core_run`, where the `clock` and `mpas_log_info` pointers are also being set.

Reply:
Yes that is the case.  JEDI calls observation operators between time steps and so needs the capability to integrate single time steps without performing a full run.

Reply:
Got it! I'll go ahead and merge this PR now.